### PR TITLE
MMCA 5042 - Implement scalafmt plugin on services V3

### PR DESCRIPTION
**Pull request by jamie2210**
- [x] Add .scalafmt.conf
- [x] Add plugins
- [x] Add scalafmtCheckAll and runAllChecks
- [x] Pass conflicts with scalastyle

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+project {
+ git = true
+}
+
+
+align {
+ preset = most
+ tokens = [ {code = "=>", owner = "Case|Type.Arg.ByName"}, "=", "<-", "->", "%", "%%", "should", "shouldBe", "shouldEqual", "shouldNot", "must" ]
```
Comment (AdamCoops) [inline .scalafmt.conf:15]: Can we break this into two lines?

Comment (jamie2210) [reply .scalafmt.conf]: Done.

**Comment by AdamCoops** [comment]
You can update the set-auto-build to 3.24.0 in the plugins file

File: .scalafmt.conf [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ arrowEnumeratorGenerator = true
+ openParenCallSite = false
+ openParenDefnSite = false
+}
+
+
+binPack {
+ parentConstructors = false
+}
+
```
Comment (AdamCoops) [inline .scalafmt.conf:26]: Do we need two empty spaces between each section on this page?

Comment (jamie2210) [reply .scalafmt.conf:23]: Removed.

**Review by AdamCoops — APPROVED** [review]

**Pull request merged**